docs/book/src/tasks/experimental-features/cluster-classes.md

```diff
@@ -0,0 +1,341 @@
+# ClusterClass
+
+The ClusterClass feature introduces a new way to create clusters which reduces boilerplate and enables flexible and powerful customization of clusters.
+ClusterClass is a powerful abstraction implemented on top of existing interfaces and offering a set of tools and operations to streamline cluster lifecycle management while maintaining the same underlying API.
+
+This tutorial covers using Cluster API and ClusterClass to create a Kubernetes cluster and how to perform a one-touch Kubernetes version upgrade.
+
+<aside class="note warning">
+
+<h1>Note</h1>
+
+This guide covers creating a ClusterClass cluster with the CAPD provider.
+</aside>
+
+## Installation
+
+### Prerequisites
+
+#### Install tools
+
+This guide requires the following tools are installed: 
+- Install and setup [kubectl] and [clusterctl]
+- Install [Kind] and [Docker]
+
+For more information see the [quick start guide].
+
+#### Enable experimental features
+
+ClusterClass is currently behind a feature gate that needs to be enabled.
+This tutorial will also use another experimental gated feature - Cluster Resource Set - in this tutorial. This is not required for ClusterClass but is used in this tutorial to set up networking.
+
+To enable these features set the respective environmental variables by running:
+```bash
+export EXP_CLUSTER_RESOURCE_SET=true
+export CLUSTER_TOPOLOGY=true
+```
+This ensures that the Cluster Topology and Cluster Resource Set features are enabled when we initialize providers.
+
+
+#### Create a CAPI management cluster
+
+A script to set up a Kind cluster pre-configured for CAPD ( the docker infrastructure provider) can be found in the [hack folder of the core repo](https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/main/hack/kind-install-for-capd.sh). 
+
+To set up the cluster from the root of the repo run:
+```bash
+./hack/kind-install-for-capd.sh
+clusterctl init --infrastructure docker
+````
+
+### Create a new Cluster using ClusterClass
+
+#### Create the ClusterClass and templates
+
+Now that we have a management Cluster with CAPD and Cluster Topology enabled we can create the ClusterClass and the templates needed to stamp clusters.
+
+The below contains six yaml objects which are used to define the cluster:
+*ClusterClass
```

Comment:
```suggestion
* ClusterClass
```
The missing space leads to:
![Screenshot 2021-10-05 at 09 04 04](https://user-images.githubusercontent.com/4662360/135975802-0e860ab6-e2f7-45b2-befa-c886d7d0088e.png)